example/generated/haberdasher_twirp.py

```diff
@@ -6,8 +6,8 @@
 from twirp.server import TwirpServer
 from twirp.client import TwirpClient
 
-from haberdasher_pb2 import Size
-from haberdasher_pb2 import Hat
+from .haberdasher_pb2 import Size
```

Comment:
We can't edit the generated file. If you need to change this, you'll need to change it in templating and test that.

Reply:
yeah, i was supposed to change the template part but forgot to do that

Reply:
I made the change in `main.go`, please check if that's the right approach

Reply:
Right now one can have the generated twirp file in any directory because the plugin generates the complete import path from the package name.

Relative import path is also fine as long as we mention it in the `README` file.

If you wish to do that, use just the file name (last element of the slice) and remove the join from the template file.